backend/ethereum/channel/subscription.go

```diff
@@ -79,7 +81,12 @@ func (a *Adjudicator) filterWatch(ctx context.Context, events chan *adjudicator.
 	}
```

Comment:
In line 57 we use the iterators error. I think it also needs an error check.  
AFAIK This applies to all iterators.